files/settings-default.cfg

```diff
@@ -1,244 +1,709 @@
+# OpenMW screen in full screen mode can be minimized when the
+# operating system regains control of the mouse and keyboard.  On
+# multiple screen configurations, disabling this option make make it
+# easier to switch between screens while playing OpenMW.
+minimize on focus loss = true
 
-inventory container x = 0
-inventory container y = 0.4275
-inventory container w = 0.6225
-inventory container h = 0.5725
+# This boolean setting determines whether frame draws are synchronized
+# with the vertical refresh rate of your monitor.  Enabling this
+# setting can reduce "tearing", a visual defect caused by updating the
+# image buffer in the middle of a screen draw.  Enabling this option
+# typically implies limiting the framerate to 60 frames per second,
+# but may also introduce additional delays caused by having to wait
+# until the appropriate time (the vertical blanking interval) to draw
+# a frame.
+
+# This setting can be adjusted in game using the "VSync" button in the
+# Video tab of the Video panel in the Options menu.  It can also be
+# changed by toggling the "Vertical Sync" check box in the Graphics
+# tab of the OpenMW Launcher.
+vsync = false
 
-inventory barter x = 0
-inventory barter y = 0.4275
-inventory barter w = 0.6225
-inventory barter h = 0.5725
+# This floating point setting controls the gamma correction for all
+# video in the game.  This setting does not currently work under
+# Linux, and the in-game setting in the Options menu has been
+# disabled.
+gamma = 1.00
 
-inventory companion x = 0
-inventory companion y = 0.4275
-inventory companion w = 0.6225
-inventory companion h = 0.5725
+[Water]
 
-container x = 0.25
-container y = 0
-container w = 0.75
-container h = 0.375
+# The water settings can be tested experimentally in the Water tab of
+# the Video panel in the Options menu.  Changes there will be saved to
+# these settings.
+
+# This boolean setting enables the refraction rendering feature of the
+# water shader.  Refraction causes deep water to be more opaque and
+# objects seen through the plane of the water to have a wavy
+# appearance.  Enabling this feature results in better visuals, and a
+# marginally lower framerate depending on your graphics hardware.  The
+# "shader" setting in the Water section must be enabled for this
+# setting to have any effect.
+refraction = false
 
-companion x = 0.25
-companion y = 0
-companion w = 0.75
-companion h = 0.375
+# Refracted texture size.  In the Video panel of the options menu, the
+# choices are Low (512), Medium (1024) and High (2048).  This setting
+# determines the resolution of the textures used for rendering objects
+# on the other wide of the plane of water (which have a wavy
+# appearance caused the by the refraction).  Higher values produces
+# better visuals and result in a marginally lower framerate depending
+# on your graphics hardware.  The "refraction" setting in the "Water"
+# section must be enabled for this setting to have any effect.
```

Comment:
'rtt size' also controls the size of the reflection texture, so "The "refraction" setting in the "Water"
+# section must be enabled for this setting to have any effect." is not correct.